### ISSUE TITLE
moondream2 on OXE Bridge dataset

### ISSUE THREAD

**Issue by zzmtsvv** [Robot-VLAs/RoboVLMs#35]
Hello,
Thank you very much for your codebase! I am concerned by the issue of running moondream2 on OXE Bridge Dataset by using your predefined scripts - the config for such a run I combined with `finetune_moondream_cont-all-lstm-post_full-ft_text_vision_wd=0_ws-8_act-10.json` (moondream2 for calvin dataset) and `finetune_kosmos_cont-lstm-post_full-ft_text_vision_wd-0_use-hand_ws-16_act-10_bridge_finetune.json` (cosmos for oxe bridge dataset). At the moment the model just does oscillation on the loss and accuracy curve - [wandb logs](https://wandb.ai/zzmtsvv/robovlms-bridge_finetune/runs/jdv47btx?nw=nwuserzzmtsvv)

maybe you  can point out the details I miss so the training would be successful?

```
{
    "robovlm_name": "RoboMoonDream",
    "parent": null,
    "task_name": "bridge_finetune",
    "model": "moondream",
    "model_url": "https://huggingface.co/vikhyatk/moondream2",
    "seq_len": 1,
    "image_size": 224,
    "image_mean": [0.48145466, 0.4578275, 0.40821073],
    "image_std": [0.26862954, 0.26130258, 0.27577711],
    "window_size": 16,
    "fwd_pred_next_n": 10,
    "arm_gripper_loss_ratio": 0.01,
    "cap_loss_ratio": 0.05,
    "fwd_loss_ratio": 0,
    "seed": 123,
    "batch_size": 1,
    "num_workers": 16,
    "data_scale": 1,
    "optimizer": "adam",
    "learning_rate": 2e-5,
    "min_lr_scale": 1e-2,
    "weight_decay": 0,
    "warmup_epochs": 0.25,
    "warmup_steps": 0,
    "warmup_ratio": null,
    "use_hand_rgb": true,
    "use_time_causal_attn": false,
    "use_mim_obs_loss": false,
    "use_pixel_loss": true,
    "use_obs_queries": true,
    "use_vision_resampler": false,
    "vision_masked_ratio": 0.9,
    "use_tube_mask": false,
    "output_root": "runs/checkpoints",
    "log_root": "runs/logs",
    "cache_root": "runs/cache",
    "model_load_path": null,
    "model_load_source": "torch",
    "resume": null,
    "model_path": ".vlms/moondream2",
    "model_config": ".vlms/moondream2/config.json",
    "train_setup": {
        "precision": "bf16",
        "predict_action": true,
        "predict_forward": false,
        "predict_forward_hand": false,
        "predict_caption": false,
        "train_vision": true,
        "bits": -1,
        "freeze_mm_mlp_adapter": false,
        "freeze_backbone": false,
        "freeze_resampler": false,
        "tune_mm_mlp_adapter": false,
        "mm_use_im_start_end": false,
        "mm_use_im_patch_token": false,
        "gradient_checkpointing": false,
        "lora_enable": false,
        "mm_projector_lr": 1e-4,
        "lora_r": 64,
        "lora_alpha": 16,
        "lora_dropout": 0.05,
        "lora_bias": "none",
        "train_text_embedding": true
    },
    "vision_resampler": {
        "vis_dim": 1024,
        "depth": 8,
        "dim_head": 64,
        "heads": 8,
        "num_latents": 64
    },
    "act_encoder": null,
    "act_head": {
        "type": "LSTMDecoder",
        "hidden_size": 1024,
        "action_dim": 7,
        "down_sample": "none",
        "latent": 1,
        "fwd_pred_next_n": 1,
        "window_size": 1,
        "action_space": "continuous",
        "with_history": true,
        "history_type": "post"
    },
    "fwd_head": null,
    "tokenizer": {
        "type": "AutoTokenizer",
        "pretrained_model_name_or_path": ".vlms/moondream2",
        "tokenizer_type": "moondream",
        "max_text_len": 256,
        "additional_special_tokens": null
    },
    "vlm": {
        "type": "AutoModelForCausalLM",
        "pretrained_model_name_or_path": ".vlms/moondream2",
        "name": "moondream"
    },
    "trainer": {
        "accelerator": "gpu",
        "strategy": "deepspeed_stage_3",
        "precision": "16",
        "logger": ["wandb"],
        "gradient_clip_val": 1.0,
        "use_distributed_sampler": false,
        "log_every_n_steps": 10,
        "max_epochs": 5,
        "val_check_interval": 10000,
        "check_val_every_n_epoch": null,
        "max_steps": 50000,
        "accumulate_grad_batches": 1
    },
    "train_dataset": {
        "type": "OpenVLADataset",
        "data_root_dir": "oxe_dataset/datasets/bridge_release/data/tfds",
        "model_name": "moondream",
        "image_aug": true,
        "mode": "train",
        "data_mix": "bridge",
        "window_sample": "sliding",
        "organize_type": "interleave",
        "shuffle_buffer_size": 51200,
        "train": true
    },
    "val_dataset": {
        "type": "OpenVLADataset",
        "data_root_dir": "oxe_dataset/datasets/bridge_release/data/tfds",
        "model_name": "moondream",
        "mode": "train",
        "data_mix": "bridge",
        "window_sample": "sliding",
        "organize_type": "interleave",
        "shuffle_buffer_size": 10000,
        "train": false
    },
    "norm_action": true,
    "norm_min": -0.65,
    "norm_max": 0.65
}

```
